src/test/suite/extensionFunctions.test.ts

```diff
@@ -270,7 +270,47 @@ suite('Extension Functions Test Suite', () => {
 		expect(content).to.not.equal(null);
 		expect(content).to.include('How do you access this amazing functionality? The Command Palette!');
 	});
+
+	test('paste from clipboard to active editor', async() => {
+		await vscode.commands.executeCommand('workbench.action.closeAllEditors');
+		await vscode.commands.executeCommand('workbench.action.files.newUntitledFile');
+		const textToPaste = 'Some text to copy into a new file created by separate command.';
+		await extensionFunctions.placeTextOnClipboard(textToPaste);
+		await extensionFunctions.pasteClipboardToActiveEditor();
+		if(vscode.window.activeTextEditor) {
+			const doc = vscode.window.activeTextEditor.document;
+			const docText = doc.getText();
+			expect(docText).to.equal(textToPaste);
+		}
+	});
 		
+	test('paste from clipboard to editor for specific file', async() => {
+		const testUriPath = 'vscode://redhat.vscode-didact?extension=src/test/data/fileToOpen.txt';
+		const testUri = handleExtFilePath(testUriPath);
+		const textToPaste = 'Some text to copy into an existing file we open.';
+		if (testUri) {
+			await extensionFunctions.placeTextOnClipboard(textToPaste);
+			await extensionFunctions.pasteClipboardToEditorForFile(testUri);
+			if(vscode.window.activeTextEditor) {
```

Comment:
the test will not fail if the edtior is not opening

Reply:
Will add an explicit fail if there's no active editor